showcase/src/assets/examples/collapsible/collapsible-custom2.html

```diff
@@ -0,0 +1,35 @@
+<stark-collapsible
+		htmlId="id0"
```

Comment:
Can you change the name of these files so it is clear what example they are related to? `collapsible-custom2` is too ambiguous...